Using devel/gtest instead of using a third party one that we
would need to download and extract.

Index: tests/CMakeLists.txt
--- tests/CMakeLists.txt.orig
+++ tests/CMakeLists.txt
@@ -1,11 +1,6 @@
 find_package(GTest REQUIRED)
 find_package(Threads REQUIRED)
 
-add_library(gtest STATIC ${GTEST_SOURCES})
-target_include_directories(gtest INTERFACE ${GTEST_INCLUDE_DIRS})
-target_include_directories(gtest PRIVATE ${GTEST_INCLUDE_DIRS} ${GTEST_SOURCE_DIR})
-target_link_libraries(gtest Threads::Threads)
-
 include_directories(
 	${CMAKE_CURRENT_SOURCE_DIR}
 )
@@ -40,6 +35,8 @@ if(BUILD_TOOLS)
 	set(SRCS ${SRCS} ../src/audio/ffmpeg_audio_reader_test.cpp)
 	link_libraries(fpcalc_libs)
 endif()
+
+link_libraries(gtest)
 
 add_executable(all_tests ${SRCS} $<TARGET_OBJECTS:chromaprint_objs>)
 target_link_libraries(all_tests PRIVATE chromaprint gtest)
